workloads/network-perf/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+source ./common.sh
+export WORKLOAD=${WORKLOAD}
+
+if [[ $WORKLOAD == "hostnet" ]]; then
+    export HOSTNETWORK=true
+elif [[ $WORKLOAD == "service" ]]; then
+    export SERVICEIP=true 
+fi
```

Comment:
else if $WORKLOAD == "pod" condition missing for pod-network-test

Reply:
what gets set from that conditional? the other workload vars changed another env but i don't see a var to change

Reply:
right